src/crypto/tls/common.go

```diff
@@ -23,9 +24,21 @@ import (
 // only supports Elliptic Curve based groups. See RFC 8446, Section 4.2.7.
 type CurveID uint16
 
+// CipherSuiteName returns the standard name for the passed cipher suite ID
+//
+// Not Implemented.
+func CipherSuiteName(id uint16) string {
+	return fmt.Sprintf("0x%04X", id)
```

Comment:
Do we want to use `strconv` or something cheaper than `fmt` here?

Reply:
I copy pasted the fallback case from the go library - also I don't think FormatUint can do leading 0s ?
so while it could be cheaper to print the value differently, at least that part could stay however big go does it?

Reply:
SGTM